.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "lib/common"]
 	path = lib/common
-	url = git@github.com:telerik/mobile-cli-lib.git
+	url = git@github.com:jasssonpet/mobile-cli-lib.git
```

Comment:
I will revert it and I will also update the change in `package.json`, once the dependencies are published/merged.